homeassistant/components/bluetooth_tracker/device_tracker.py

```diff
@@ -128,21 +144,26 @@ async def async_setup_scanner(
     if request_rssi:
         _LOGGER.debug("Detecting RSSI for devices")
 
-    async def perform_bluetooth_update():
+    async def perform_bluetooth_update() -> None:
         """Discover Bluetooth devices and update status."""
         _LOGGER.debug("Performing Bluetooth devices discovery and update")
         tasks = []
```

Comment:
```suggestion
        tasks: list[asyncio.Task] = []
```

Reply:
now I get `error: Argument 1 to "append" of "list" has incompatible type "Coroutine[Any, Any, None]"; expected "Task[Any]"  [arg-type]` which reference to
https://github.com/home-assistant/core/blob/b494c570a0c6b84d47998f5f5fad50634ebe5485/homeassistant/components/bluetooth_tracker/device_tracker.py#L176-L178

Reply:
Right, it's actually a `Coroutine`. `Awaitable[None]` should also work.